subsys/net/lib/download_client/src/download_client.c

```diff
@@ -274,6 +275,40 @@ static int get_request_send(struct download_client *client)
 	return 0;
 }
 
+static int my_strncasecmp(const char *s1, const char *s2, size_t n)
+{
+	if (n != 0) {
+		do {
+			if (tolower(*s1) != tolower(*s2++))
+				return (tolower(*s1) - tolower(*--s2));
```

Comment:
Zephyr coding standard requires to add braces even for single-line code blocks. Please fix wherever applicable.